packages/dataviews/src/view-list.tsx

```diff
@@ -396,13 +415,14 @@ export default function ViewList< Item >( props: ViewListProps< Item > ) {
 						key={ id }
 						id={ id }
 						actions={ actions }
+						fields={ fields }
 						item={ item }
 						isSelected={ item === selectedItem }
 						onSelect={ onSelect }
 						mediaField={ mediaField }
 						primaryField={ primaryField }
 						store={ store }
-						visibleFields={ visibleFields }
+						defaultFields={ defaultFields }
```

Comment:
We could remove this prop and only operate with `fields={ otherFields }`, see [this comment](https://github.com/WordPress/gutenberg/pull/63148/files#r1666512418).